evalml/tests/automl_tests/test_dask_engine.py

```diff
@@ -154,8 +154,9 @@ def eval_pipelines(pipelines, engine):
         assert all([s._is_fitted for s in par_pipelines])
 
         # Ensure the scores in parallel and sequence are same
-        assert set(par_scores) == set(seq_scores)
```

Comment:
This test was flaking on me in the 17th decimal place, but only for [linux](https://github.com/alteryx/evalml/pull/2297/checks?check_run_id=2635860715)/[windows](https://github.com/alteryx/evalml/pull/2297/checks?check_run_id=2635667949) python 3.8 😆  not sure why exactly this PR would trigger that.